client/packages/system/src/Manage/Reports/hooks/useAllReportVersionsList.ts

```diff
@@ -0,0 +1,102 @@
+import {
+  useQuery,
+  SortBy,
+  ReportSortFieldInput,
+  ReportFilterInput,
+  useIntlUtils,
+  useTranslation,
+  useNotification,
+  LIST_KEY,
+} from '@openmsupply-client/common';
+import {  ALLREPORTVERSIONS } from './keys';
+import { ReportRowFragment } from 'packages/system/src/Report/index.js';
+import { useAllReportVersionsGraphQL } from '../api/useReportsGraphQL';
+
+export type ReportListParams = {
+  filterBy: ReportFilterInput | null;
+  sortBy?: SortBy<ReportRowFragment>;
+  first?: number;
+  offset?: number;
+};
+
+export const useAllReportVersionsList = ({
+  queryParams,
+}: {
+  queryParams?: ReportListParams;
+}) => {
+
+    // QUERY
+
+    const { data, isLoading, isError } = useGetList(queryParams);
+
+    return {
+        query: {data, isLoading, isError},
+    };
+};
+
+const useGetList = (queryParams?: ReportListParams) => {
+  const { reportApi, storeId } = useAllReportVersionsGraphQL();
+  const { currentLanguage: language } = useIntlUtils();
+  const { error } = useNotification();
+  const t = useTranslation();
```

Comment:
NIT: would personally move `t` function to the top of hook. Its an important hook, though IMO it doesn't have a major value while reading the overall business logic of the hook

Reply:
This is actually already following the rule of thumb for React where hooks are at the top. In terms of order, well, it's really just preference at this point 😁 haha. 

Though have found some other devs move the `t` function to the top as side quests as well 👀